File: URI/Ruby/uri-1013.rb
#URI - 1013: https://www.urionlinejudge.com.br/judge/pt/problems/view/1013

Input = gets.split(' ')
A = Input[0].to_i
B = Input[1].to_i
C = Input[2].to_i
BiggerAB = (A + B + (A - B).abs) / 2

if BiggerAB > C
    puts "#{BiggerAB} eh o maior"
else
    puts "#{C} eh o maior"
end

File: URI/C#/uri-1013.cs
// URI - 1013

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] values = Console.ReadLine().Split(' ');

            HighestValue(values);
        }

        private static void HighestValue(string[] values)
        {
            int aValue = int.Parse(values[0]);
            int bValue = int.Parse(values[1]);
            int cValue = int.Parse(values[2]);

            int greaterBetweenAandB = ((aValue + bValue + Math.Abs(aValue - bValue)) / 2);

            if (greaterBetweenAandB < cValue)
            {
                Console.WriteLine(cValue + " eh o maior");
            }
            else
            {
                Console.WriteLine(greaterBetweenAandB + " eh o maior");
            }
        }
    }
}